content/en/providers/check-in/sp/_index.md

```diff
@@ -698,81 +757,105 @@ user authorisation:
 
-#### Configure simple-oidc-client
+Now that you have everything you need, you can configure your login settings. Go
+to your terminal and open `config.php` with your favorite text editor.
 
-Now that we have everything we need, we can configure our login settings. Go to
-your terminal and open `configuration.js` with your favorite text editor. ex.
+Example:
 
 ```shell
-vi simple-oidc-client/configuration.js
+vi simple-oidc-client-php/config.php
 ```
 
 Let's go quickly through the settings:
 
-- `title` is the title on the navigation bar.
-- `authority` is the base URL of our IdentityServer instance. This will allow
-  oidc-client to query the metadata endpoint so it can validate the tokens.
-- `client_id` is the id of the client we want to use when hitting the
-  authorization endpoint.
-- `popup_redirect_uri` is the redirect URL used when using the signinPopup
-  method. If you prefer not having a popup and redirecting the user in the main
-  window, you can use the `redirect_uri` property and the signinRedirect method.
-- `post_logout_redirect_uri` is the redirect URL used when using the
-  signoutRedirect method.
-- `response_type` defines in our case that we only expect an identity token
-  back.
-- `scope` defines the scopes the application asks for.
-- `debug` displays user data.
-- `filterProtocolClaims` indicates to oidc-client if it has to filter some OIDC
-  protocol claims from the response: `nonce`, `at_hash`, `iat`, `nbf`, `exp`,
-  `aud`, `iss` and `idp`.
-
-You must change the followings options based on your client configuration:
-
-- `authority` (issuer)
+- `title` required, is the title on the navigation bar
+- `img` required, is the source of the logo
+- `scope_info` optional, is a message that informs the user for the application
+  requirements
+- `issuer` required, is the base URL of our IdentityServer instance. This will
+  allow oidc-client to query the metadata endpoint so it can validate the tokens
+- `client_id` required, is the id of the client we want to use when hitting the
+  authorization endpoint
+- `client_secret` optional, a value the offers better security to the message
+  flow
+- `pkceCodeChallengeMethod` optional, a string that defines the code challenge
+  method for PKCE. Choose between `plain` or `S256`.
+- `redirect_url` required, is the redirect URL where the client and the browser
+  agree to send and receive correspondingly the code
+- `scopesDefine` required, defines the scopes the client supports
+- `refresh_token_note` optional, info for the refresh token
+- `access_token_note` optional, info for the access token
+- `manage_token_note` optional, message the informs the user where can manage
+  his tokens
+- `manageTokens` optional, URL of the manage tokens service
+- `sessionName` required, define the name of the cookie session
+- `sessionLifetime` required, define the duration of the session. This must be
+  equal to the validity time of the access token.
```

Comment:
```suggestion
- `title` required, the title on the navigation bar
- `img` required, the source of the logo
- `scope_info` optional, a message that informs the user for the application
  requirements
- `issuer` required, the base URL of our IdentityServer instance. This will
  allow oidc-client to query the metadata endpoint so it can validate the tokens
- `client_id` required, the id of the client we want to use when hitting the
  authorization endpoint
- `client_secret` optional, a value the offers better security to the message
  flow
- `pkceCodeChallengeMethod` optional, a string that defines the code challenge
  method for PKCE. Choose between `plain` or `S256`.
- `redirect_url` required, the redirect URL where the client and the browser
  agree to send and receive correspondingly the code
- `scopesDefine` required, defines the scopes the client supports
- `refresh_token_note` optional, info for the refresh token
- `access_token_note` optional, info for the access token
- `manage_token_note` optional, message the informs the user where can manage
  his tokens
- `manageTokens` optional, URL of the manage tokens service
- `sessionName` required, define the name of the cookie session
- `sessionLifetime` required, define the duration of the session. This must be
  equal to the validity time of the access token.
```

Reply:
Resolved by commit [Fix typos](https://github.com/EGI-Federation/documentation/pull/332/commits/7c9c7855873f8858ed9f14a4a5119ff667c32741)